api/schema/rvinfo.yaml

```diff
@@ -0,0 +1,234 @@
+          minimum: 2
+          maximum: 2
+        RVValue:
+          oneOf:
+            - type: string
+              format: ipv4
+            - type: string
+              format: ipv6
+    RVDevPort:
+      description: Protocol specific port number used by Device
+      type: object
+      properties:
+        RVVariable:
+          type: integer
+          format: uint8
+          minimum: 3
+          maximum: 3
+        RVValue:
+          type: integer
+          format: uint16
+    RVOwnerPort:
+      description: Protocol specific port number used by Owner
+      type: object
+      properties:
+        RVVariable:
+          type: integer
+          format: uint8
+          minimum: 4
+          maximum: 4
+        RVValue:
+          type: integer
+          format: uint16
+    RVDns:
+      description: DNS address
+      type: object
+      properties:
+        RVVariable:
+          type: integer
+          format: uint8
+          minimum: 5
+          maximum: 5
+        RVValue:
+          type: string
+          format: hostname
+    RVSvCertHash:
+      description: TLS server certificate hash
+      type: object
+      properties:
+        RVVariable:
+          type: integer
+          format: uint8
+          minimum: 6
+          maximum: 6
+        RVValue:
+          type: object
+          properties:
+            HashType:
+              type: integer
+            Hash:
+              type: string
+              format: byte
+    RVClCertHash:
+     description: TLS Client CA certificate hash
+     type: object
+     properties:
+       RVVariable:
+         type: integer
+         format: uint8
+         minimum: 7
+         maximum: 7
+       RVValue:
+         type: object
+         properties:
+           HashType:
+             type: integer
+           Hash:
+             type: string
+             format: byte
+    RVUserInput:
+      description: Use user input if available
+      type: object
+      properties:
+        RVVariable:
+          type: integer
+          format: uint8
+          minimum: 8
+          maximum: 8
+        RVValue:
+          type: boolean
+    RVWifiSsid:
+      description: WiFi SSID of the onboarding network
+      type: object
+      properties:
+        RVVariable:
+          type: integer
+          format: uint8
+          minimum: 9
+          maximum: 9
+        RVValue:
+          type: string
+    RVWifiPw:
+      description: Password for access to RVWifiSsid network
+      type: object
+      properties:
+        RVVariable:
+          type: integer
+          format: uint8
+          minimum: 10
+          maximum: 10
+        RVValue:
+          type: string
+    RVMedium:
+      description: Preferred network interface
+      type: object
+      properties:
+        RVVariable:
+          type: integer
+          format: uint8
+          minimum: 11
+          maximum: 11
+        RVValue:
+          type: integer
+          format: uint8
+    RVProtocol:
```

Comment:
We might want to add more information about the RVProtocolValue:
``` 
RVProtocolValue /= (
    RVProtRest    => 0,
    RVProtHttp    => 1,
    RVProtHttps   => 2,
    RVProtTcp     => 3,
    RVProtTls     => 4,
    RVProtCoapTcp => 5,
    RVProtCoapUdp => 6
);
```

See:
https://stackoverflow.com/questions/66465888/how-to-define-enum-mapping-in-openapi and https://github.com/OAI/OpenAPI-Specification/issues/348#issuecomment-336194030

Reply:
I've updated the PR with a better description of RVProtocolValue, RVMediumValue, and TransportProtocol.  Includes some fixes to issue flagged by redocly.  I've also added at github action to run the linter on the schema.